tests/buildtest/crates/missing-deps-registry-version/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "missing-deps-registry-version"
+version = "0.1.0"
+authors = ["Pietro Albini <pietro@pietroalbini.org>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+empty-library = "=0.5.0"
```

Comment:
can you explain what the probem here is?

Reply:
[A version 0.5 does not exist on crates.io](https://crates.io/crates/empty-library/versions).

This is to reproduce an error like <https://crater-reports.s3.amazonaws.com/beta-1.86-1/1.85.0/gh/adelelopez.bayes/log.txt>